src/pages/solution/adobe-campaign-classic/tabs/push-payloads.md

```diff
@@ -0,0 +1,161 @@
+| Icon | ⛔️ | `adb_icon` | string | URI or name for Large icon used in notification. |
+| Channel ID | ⛔️ | `adb_channel_id` | string | The [notification's channel id](https://developer.android.com/guide/topics/ui/notifiers/notifications#ManageChannels) (new in Android O). The app must create a channel with this channel ID before any notification with this channel ID is received. If you don't send this channel ID in the request, or if the channel ID provided has not yet been created by the app, FCM uses the channel ID specified in the app manifest.<br />If not provided in payload, SDK uses a "default" channel ID of value "CampaignPushChannel".<br />If < API 26 (Android O), this value is ignored. |
+| Badge Count | ⛔️ | `adb_n_count` | string | Value to be used on app's badge. |
+| Priority | ⛔️ | `adb_n_priority` | string | Designates the notification priority for Android - [importance](https://developer.android.com/reference/android/app/NotificationChannel#setImportance(int)) for API >= 26 (Android 8.0) or [priority](https://developer.android.com/reference/androidx/core/app/NotificationCompat.Builder#setPriority(int)) for API < 26 |
+| Tag | ⛔️ | `adb_tag` | string | Identifier used to replace existing notifications in the notification drawer. If not specified, each request creates a new notification. If specified and a notification with the same tag is already being shown, the new notification replaces the existing one in the notification drawer. |
+| Sticky | ⛔️ | `adb_sticky` | string | Boolean value represented as a string "true" or "false".<br />When set to false or unset, the notification is automatically dismissed when the user clicks it in the panel. When set to true, the notification persists even when the user clicks it. |
+| Ticker | ⛔️ | `adb_ticker` | string | Sets the "ticker" text, which is sent to accessibility services. Prior to API level 21 (Lollipop), sets the text that is displayed in the status bar when the notification first arrives. |
+| Color - Small Icon | ⛔️ | `adb_clr_icon` | string | Color for the notification's small icon.<br />Represented as six character hex, e.g. `00FF00` |
+| Color - Background | ⛔️ | `adb_clr_bg` | string | Color for notification's background.<br />Represented as six character hex, e.g. `00FF00` |
+
+### Example
+
+Below is a sample of what a payload might look like for a carousel notification:
+
+```json
+{
+	"message": {
+		"android": {
+			"collapse_key": "new_message",
+			"priority": "HIGH",
+			"data": {
+				"adb_version": "1",
+                "adb_template_type": "car",
+				"adb_title": "Check out the new line of shoes!",
+				"adb_body": "Hot deals on new releases!",
+				"adb_sound": "sneakerSqueaker",
+				"adb_uri": "https://sneakerland.com/newReleases",
+				"adb_a_type": "WEBURL",				
+		        "adb_body_ex": "We have some new inventory we think you'll like.",
```

Comment:
i'm seeing some red background added to this code sample here and at https://github.com/AdobeDocs/aep-mobile-sdkdocs/pull/477/files#diff-94bef8f79c929b7980d99f27b255e44d1cb7753f3322b0c2d462f5b5fb8243caR20. not sure if its just my browser though.

Reply:
weird github issue - looks good rendered